observer/lib/observer.js

```diff
@@ -129,3 +129,32 @@ export const observeRetrievalResultCodes = async (pgPoolStats, influxQueryApi) =
     ])
   }
 }
+
+export const observeDesktopUsers = async (pgPoolStats, influxQueryApi) => {
+  const rows = await influxQueryApi.collectRows(`
+    from(bucket: "station-machines")
+      |> range(start: -24h)
```

Comment:
Do you have any concerns about this query being defined as over the last 24h? This means depending on when this function is run, you will get a different count for the day. It would be safer to query the full 24h from the last day, always, for reproducible results.

Reply:
Good point, I think it would make sense to query results for previous day with exact start and end time.

Reply:
Found a really nice, clean way to do it using [boundaries.yesterday](https://docs.influxdata.com/flux/v0/stdlib/experimental/date/boundaries/yesterday/) function, only problem is that it's part of the experimental package. What do you think, should we use that or have it passed down from javascript?